sockeye/inference.py

```diff
@@ -880,25 +944,31 @@ def _beam_search(self,
 
         return sequences, attentions, scores_accumulated, lengths
 
-    @staticmethod
-    def _get_best_from_beam(sequences: mx.nd.NDArray,
+    def _get_best_from_beam(self,
+                            sequences: mx.nd.NDArray,
                             attention_lists: mx.nd.NDArray,
                             accumulated_scores: mx.nd.NDArray,
-                            lengths: mx.nd.NDArray) -> Translation:
+                            lengths: mx.nd.NDArray) -> List[Translation]:
         """
         Return the best (aka top) entry from the n-best list.
 
-        :param sequences: Array of word ids. Shape: (beam_size, bucket_key).
+        :param sequences: Array of word ids. Shape: (batch_size * beam_size, bucket_key).
         :param attention_lists: Array of attentions over source words. Shape: (length, bucket_key).
```

Comment:
can you update the shape comment? Should be `(batch_size * self.beam_size, max_output_length, encoded_source_length)` no?

Reply:
fixed